mtgjson5/providers/cardkingdom.py

```diff
@@ -178,11 +175,15 @@ def update_sealed_product(
 
             sealed_product.identifiers.card_kingdom_id = str(product["id"])
 
-            sealed_product.category = sealed_product.determine_mtgjson_sealed_product_category(
-                sealed_product.name.lower()
+            sealed_product.category = (
```

Comment:
Did tox clean this file up here? Feels unmodified

Reply:
Yeah this was a tox change and I didn't question it. I can remove these if you want.